### PR TITLE
Chat1 1 version2

### PR DESCRIPTION
**Pull request by bonnieee702**

### CODE REVIEW

File: backend/static/backend/css/chatindex_style.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.chat-sidebar {
+    width: 250px;
+    background-color: #f7f7f7;
+    height: 100vh;
+    overflow-y: auto;
+  }
+  
+  .chat-user-link {
```
Comment (Vialor) [inline backend/static/backend/css/chatindex_style.css:8]: [nits] Just a formatting issue
Can u remove the space before the selectors?

File: backend/static/backend/css/chatwithuser_style.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/* Chat Container Styles */
+.chat-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between; /* Aligns children (chat-box, form) vertically */
+    height: 100%; /* Full height of the container */
+  }
+  
+  /* Chat Box Styles */
+  #chat-box {
```
Comment (Vialor) [inline backend/static/backend/css/chatwithuser_style.css:10]: Same as above

File: backend/views/chatHandler.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# views.py
```
Comment (Vialor) [inline backend/views/chatHandler.py:1]: Can u use this format: `chat_handlers.py` ?

**Review by Vialor — APPROVED** [review]
Thanks for your hard work, but pls fix the styling issue.
Also like mentioned last time, it might not be a good idea to include secret and password in github, but I guess it is ok for such a small project.

**Comment by bonnieee702** [comment]
> Thanks for your hard work, but pls fix the styling issue. No need for re-review. Also like mentioned last time, it might not be a good idea to include secret and password in github, but I guess it is ok for such a small project.

thanks for the review! just fixed the styling. And for the pusher secret, it's a very good point, i was think about add it to github secret before, but i don't think we has access for that feature, maybe i can add it to travis. I will think about it, thanks!

**Pull request merged**